resources/shaders/shader.frag

```diff
@@ -13,29 +13,36 @@ out vec4 color;
 // white light
 vec3 lightColor = vec3(1.0, 1.0, 1.0);
 
+struct Material {
+    vec4 ambient;
+    vec4 diffuse;
+    vec4 specular;
+    float shininess;
```

Comment:
What is the difference between the two?

Reply:
shininess vs shininess_strength

Feels like if they want the strength large just make the shininess larger?

Reply:
According to the docs:

SHININESS
Defines the shininess of a phong-shaded material. This is actually the exponent of the phong specular equation	
SHININESS=0 is equivalent to SHADING_MODEL=aiShadingMode_Gouraud.

SHININESS_STRENGTH	
Scales the specular color of the material.